src/controllers/expensesController.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getExpenseById,
+  createExpense,
+  removeExpense,
+  updateExpense,
+} = require('../services/expensesService');
+
+const { getUserById } = require('../services/usersService');
+
+function getExpenses(req, res) {
+  const { userId, categories, from, to } = req.query;
+
+  const searchedExpenses = getAllExpenses(
+    userId,
+    categories,
+    from,
+    to,
+  );
```

Comment:
```suggestion
  const searchedExpenses = getAllExpenses(req.query);
```
This could be simplified in this case